package.json

```diff
@@ -115,7 +115,7 @@
     "cross-env": "^2.0.0",
     "css-loader": "^0.25.0",
     "ejs": "^2.5.1",
-    "enzyme": "^2.4.1",
+    "enzyme": "^2.8.2",
```

Comment:
Note: This is just a minimum version change, previously if you did a fresh npm install, you would be able to get this version. This just forces it to be at least this version as enzyme fixed some things in this version to allow it to work with react 15.5